src/mountutils.cpp

```diff
@@ -22,4 +24,12 @@ NAN_MODULE_INIT(MountUtilsInit) {
   .ToLocalChecked());
 }
 
+
+void MountUtilsLog(std::string string) {
+  const char* debug = std::getenv("MOUNTUTILS_DEBUG");
```

Comment:
I was thinking about following the environment variable convention we have in Etcher, where variables start with `ETCHER_`, but I have no strong opinions about it.